packages/GenderSelection/src/GenderSelection.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, AllHTMLAttributes, FC, RefObject, RefAttributes, useRef, useEffect} from 'react';
+import classnames from 'classnames';
+import styles from './GenderSelection.scss';
+import { Input } from "@mbkit/input";
+import { Select } from "@mbkit/select";
+import { Label } from "@mbkit/label";
+
+export type SelectOptions = {
+    label: string;
+    value: string;
+}[]
+
+export type GenderProps = AllHTMLAttributes<HTMLDivElement> &
+    RefAttributes<HTMLDivElement> & {
+        // return object on change of selection 
+        onChange: (event: any) => void;
+
+        // options for select 
+        options: SelectOptions;
```

Comment:
Since we're passing in `options` and `onChange` let's also pass in `value` and make this a fully controlled component.

Reply:
Meaning, let's not hold state within it and just pass in the props needed to manage it

Reply:
Made as Fully controlled component